content/ja/docs/reference/kubectl/_index.md

```diff
@@ -66,6 +66,32 @@ kubectl [command] [TYPE] [NAME] [flags]
 
 ヘルプが必要な場合は、ターミナルウィンドウから`kubectl help`を実行してください。
 
+## クラスター内認証と名前空間のオーバーライド {#in-cluster-authentication-and-namespace-overrides}
+
+デフォルトでは、`kubectl`は最初にPod内で動作しているか、つまりクラスター内で動作しているかを判断します。まず、`KUBERNETES_SERVICE_HOST`と`KUBERNETES_SERVICE_PORT`の環境変数を確認し、サービスアカウントのトークンファイルが`/var/run/secrets/kubernetes.io/serviceaccount/token`に存在するかどうかを確認します。3つともクラスター内で見つかった場合、クラスター内認証とみなされます。
+
+後方互換性を保つため、クラスター内認証時に`POD_NAMESPACE`環境変数が設定されている場合には、サービスアカウントトークンのデフォルトの名前空間が上書きされます。名前空間のデフォルトに依存しているすべてのマニフェストやツールは、この影響を受けます。
+
+**`POD_NAMESPACE`環境変数**
+
+`POD_NAMESPACE`環境変数が設定されている場合、名前空間に属するリソースのCLI操作は、デフォルトで環境変数の値になります。これは、Podが名前空間に属するリソースであり、コマンドで名前空間が指定されていないためです。`kubectl api-resources`の出力を見て、リソースが名前空間に属するかどうかを判断してください。
```

Comment:
There is the sentence of followings on [`en` content](https://deploy-preview-40138--kubernetes-io-main-staging.netlify.app/docs/reference/kubectl/):
```
For example, if the variable is set to seattle, kubectl get pods would return pods in the seattle namespace. 
```

It seems the sentence is not translated into `ja`.

Reply:
That sentence seems to had been forgotten.
It was translated and added in ja.